app/src/main/java/org/helfoome/presentation/restaurant/RestaurantMenuAdapter.kt

```diff
@@ -0,0 +1,70 @@
+package org.helfoome.presentation.restaurant
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import org.helfoome.databinding.ItemMenuBinding
+import org.helfoome.databinding.ItemMenuNutrientBinding
+import org.helfoome.domain.entity.MenuInfo
+
+class RestaurantMenuAdapter : RecyclerView.Adapter<RecyclerView.ViewHolder>() {
+    private var menuCardViewType = MenuCardViewType.MENU_VIEW_TYPE
+    private var menuList = listOf<MenuInfo>()
```

Comment:
프로퍼티 이용해서 getter, setter 메소드 안쓰고 좀 더 깔끔쓰하게 표현가능합니다~! 참고!
```suggestion
    private val _menuList = mutableListOf<MenuInfo>()
    val menuList : List<ResGoalGet.Data.Result.Keyword> = _menuList
        set(value) {
            _menuList.clear()
            _menuList.addAll(value)
            notifyDataSetChanged()
        }
```